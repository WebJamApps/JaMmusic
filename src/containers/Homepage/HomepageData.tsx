/* eslint-disable react/jsx-one-expression-per-line */

// function Faqs(): JSX.Element {
//   return (
//     <div className="faqs">
//       <h3>FAQs</h3>
//       <div className="qAnda">
//         <p><span>Q: </span>How can I contact you?</p>
//         <p><span>A: </span>Send an email to Josh &amp; Maria Sherman
//           <br />
//           <a href="mailto:web.jam.adm@gmail.com"><strong>web.jam.adm@gmail.com</strong></a>
//           <br />
//           or call&nbsp;
//           <a href="tel:5404948035"><strong>540-494-8035</strong></a>
//         </p>
//         <br />
//         <p><span>Q: </span>What does Web Jam LLC do?</p>
//         <p><span>A: </span>We provide mobile-friendly web technologies and services.</p>
//         <br />
//         <p><span>Q: </span>What is your mission statement?</p>
//         <p>
//           <span>A: </span>
//           To assist small businesses by refactoring their existing
//           websites to be mobile-friendly,
//           providing new online services, empowering business owners, and training staff to use these new services
//           effectively.
//         </p>
//       </div>
//     </div>
//   );
// }

// export default Faqs;
