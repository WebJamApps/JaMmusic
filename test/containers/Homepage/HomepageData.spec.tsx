
import { shallow } from 'enzyme';
import HomepageData from '../../../src/containers/Homepage/HomepageData';

const wrapper = shallow<typeof HomepageData>(<HomepageData />);

describe('HomepageData', () => {
  it('Renders the homepageData', () => {
    expect(wrapper.find('div.qAnda').exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
