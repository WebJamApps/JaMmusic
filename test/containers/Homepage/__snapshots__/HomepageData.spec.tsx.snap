// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomepageData Renders the homepageData 1`] = `ShallowWrapper {}`;
